openmicroscopy/openmicroscopy
# Rnd rgb

Set the min/max to 0-255 for 8-bit graphics images.
I have limited it: png, jpeg, bmp, gif and Tiff
The issue has been reported by users on Forum.

to test:
- Import bmp image from test_image_good.  Before one channel was set to 0-128. Make sure it is now set to 0-255
  ![rgbbefore](https://cloud.githubusercontent.com/assets/1022396/4823108/58474ac6-5f4b-11e4-9159-3ee91e1cf0f9.png)
  ![rgbafter](https://cloud.githubusercontent.com/assets/1022396/4823111/5f618984-5f4b-11e4-8e98-55cacaa6ddb5.png)
